src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -74,6 +91,27 @@ def map_alternatives_items
       end
     end
 
+    def filtered_alternatives
+      alternatives = @alternatives_list.select { |a| Regexp.new(@search).match(a.name) }
```

Comment:
another options is to use `include?` like in https://github.com/yast/yast-cio/blob/master/src/lib/iochannel/channels_dialog.rb#L154

so when you want vim you can just write `vi` and it show everything with vi. thats how usually quick search works. Not so many users knows ruby regexps.